lib/main.js

```diff
@@ -79,4 +83,22 @@ module.exports = {
 
 }
 
+function _findFileInParents(path, encoding, silent) {
+  for (var i = 0; i < 100; i++) {
+    try{
+      fs.readFileSync(path, { encoding: encoding })
+      return path
+    }
+    catch (e){
+      path = '../' + path
```

Comment:
I would like to see more demand from the community and a complete PR including tests and documentation before seriously considering this addition